pyphare/pyphare/mock_mhd_simulator/simulator.py

```
@@ -0,0 +1,48 @@
+from . import clearDict
+from . import populateDict
+
+def pyMHD():
+    import importlib
+
+    return importlib.import_module("pybindlibs.pyMHD")
+
+def make_cpp_simulator(dim, interp):
+    import pybindlibs.pyMHD
```

Ruff (0.8.2)

`pybindlibs.pyMHD` imported but unused

Remove unused import: `pybindlibs.pyMHD`

(F401)